docs/how_we_work.md

```diff
@@ -30,6 +30,29 @@
 * *What*:	We will write each pull request to accomplish one small, cohesive change. This change will map back to an issue for reference. We will include a link to the associated issue for each pull request. We may need more than one pull request to implement a given issue.
 * *Why*:	Small and focused pull requests will make code review faster and easier.  If there’s a need to roll back a change, it can also be done more easily if we do not have a big change or many different changes in a pull request.  By mapping pull requests back to issues, it can help the team track our progress on our work.
 
+## Engineering team roles
+
+### Tech Lead role
+
+Amy Mok (AM) will act as our Tech Lead for this project, with backup support from Alex Soble (ARS) and Carly Jugler (CJ).
+
+### Table of responsibilities
+
+Task/Responsibility | Primary | Backup
+-- | -- | --
+Keep tabs on the different eng streams and whether they’re flowing smoothly individually and together<br>
```

Comment:
```suggestion
Keep tabs on the different eng streams and whether they’re flowing smoothly individually and together | AM | ARS
```

In the preview it looks like the first row of the table is blank; this suggestion would fill in that role.